app/controllers/stats_controller.rb

```diff
@@ -163,10 +163,17 @@ def create_spreadsheet(report)
     axlsx
   end
 
+  def is_general_close_report_present?
+    close_report_scope = Pundit.policy_scope(current_user, ReportType.closed_cases_report)
+    close_report_scope.count > 0
+  end 
+
   def user_permitted_custom_report_types
     # find out the scope of the custom report types the user can see via set intersection operation
     @correspondence_types = CorrespondenceType.custom_reporting_types & current_user.permitted_correspondence_types
-    @correspondence_types += [self.class.closed_cases_correspondence_type]
+    if is_general_close_report_present?
+      @correspondence_types += [self.class.closed_cases_correspondence_type]
```

Comment:
I find the += [...] a bit confusing here. Are we always going to get an Array out of line 173? Then we should just be able to do

```suggestion
      @correspondence_types << self.class.closed_cases_correspondence_type
```

Reply:
If @correspondence_types comes out as `nil`, the above code blows up anyway :) 

Could we make it the `<<` just for ease of reading?